context/app/static/js/components/Header/HeaderContent/HeaderContent.jsx

```diff
@@ -27,10 +44,15 @@ function HeaderContent({ anchorRef }) {
       {!shouldDisplayMenu && (
         <>
           <FlexNoWrap>
-            {['Donor', 'Sample', 'Dataset'].map((type) => (
-              <HeaderButton key={type} href={`/search?entity_type[0]=${type}`} component={Link}>
-                {`${type}s`}
-              </HeaderButton>
+            {types.map((type) => (
+              <HeaderType>
+                <div>
```

Comment:
If you remove this `div` and the `margin-top` in `StyledSvgIcon` the appearance is the same.